### PR TITLE
Tries to add learning_materials resource

### PR DESCRIPTION
**Pull request by roosdebildt**
I'm trying to add learning materials as a resource and a controller in our app. I'm not all the way there yet, but I got stuck on my way.

### CODE REVIEW

**Comment by roosdebildt** [comment]
I'm trying to add learning materials as a resource and a controller in our app. I'm not all the way there yet, but I got stuck on my way. I get this error:

screen shot 2015-08-05 at 14 57 09

@nathanvda any idea why?

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,18 @@
 Rails.application.routes.draw do
+
+  get 'learning_materials/index'
+
+  get 'learning_materials/new'
+
+  get 'learning_materials/show'
```
Comment (yorickpeterse) [inline config/routes.rb]: These 3 routes are automatically created by `resources :learning_materials`.

**Comment by yorickpeterse** [comment]
At least in Rails a POST request is usually sent to a URL without `/new` or `/edit` in it, so something like a POST to `/learning_materials` instead of `/learning_materials/new`.

Interesting enough there _should_ be a route that handles POST requests sent to `/learning_materials` (due to `resources :learning_materials` being present in `config/routes.rb`), but for some reason it's not, nor is the form sending requests to it :/

**Comment by roosdebildt** [comment]
we found the bug!

**Pull request merged**